src/dlgAboutDialog.cpp

```diff
@@ -255,10 +262,11 @@ void dlgAboutDialog::setAboutTab(const QString& htmlHead) const
            "<p>Special thanks to <span style=\"color:#bc8942;\"><b>Nick Gammon</b></span> (<a href=\"http://www.gammon.com.au/mushclient/mushclient.htm\">www.gammon.com.au/mushclient/mushclient.htm</a>) for giving us some valued pieces of advice.</p>"));
 
     textBrowser_mudlet->setHtml(
-            QStringLiteral("<html>%1<body><table border=\"0\" style=\"margin-top:36px; margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
+            QStringLiteral("<html>%1<body><table border=\"0\" style=\"margin:18px 36px; width: 100%;\" cellspacing=\"2\" cellpadding=\"0\">\n"
```

Comment:
```suggestion
            QStringLiteral("<html>%1<body><table border=\"0\" style=\"margin-top:18px; margin-bottom:18px; margin-left:36px; margin-right:36px; width: 100%;\" cellspacing=\"2\" cellpadding=\"0\">\n"
```

Reply:
Horizontal is as it was. First value is top + bottom, second is left and right.

Reply:
I'm sorry but I don't think that is working - it looks like there is no margins at the sides - look how close `Documentation` is to `wiki.mudlet.org/w/Main_page` - if you refer to https://doc.qt.io/qt-5/richtext-html-subset.html#css-properties you might see that Qt's rich-text engine only supports single values - remember this is Qt's own rich text and not actual HTML and although similar it is not an exact match - or rather it does not support some of the more modern short-cuts...

Reply:
Good visual catch! Although I got to greedy! `width` is not supported, margins looks OK. I guess QT docs are missing them (since padding shorthand is there and they are working).

Reply:
![image](https://user-images.githubusercontent.com/3740628/103325841-e92e6080-4a4d-11eb-81fe-d8195ac9eb1b.png)
 Curious thing though, when building using command line and qmake or CLion git properties are populated, when I did run it now from VS Code it's not. But I guess it's not that much of importance right now. Will try to see why it happens later (I guess my config issue).